cli/main.py

```diff
@@ -219,6 +219,16 @@ def show_examples():
             "TAVILY_API_KEY",
         ],
     },
+    ToolName.BraveSearch: {
+        "secrets": [
+            "BRAVE_SEARCH_API_KEY",
+        ],
+    },
+    ToolName.WolframAlpha: {
+        "secrets": [
+            "WOLFRAM_ALPHA_API_KEY",
```

Comment:
maybe `WOLFRAM_ALPHA_APP_ID` instead to be in continuity with the env name and lib naming?